data_scrapers/alameda_county.py

```diff
@@ -0,0 +1,196 @@
+    Note that the DataFrame will convert the "<10" strings to NaN.
+    """
+    # Dicts of target_label : source_label for re-keying. 
+    # Note that the cases table includes MTF and FTM, but the deaths table does not. 
+    GENDER_KEYS = {"female": "Female", "male": "Male",
+                   "unknown": "Unknown_Sex", "mtf": "MTF", "ftm": "FTM"} 
+    RACE_KEYS = {"Latinx_or_Hispanic": "Hispanic_Latino", "Asian": "Asian", "African_Amer": "African_American_Black",
+                 "White": "White", "Pacific_Islander": "Pacific_Islander", "Native_Amer": "Native_American", "Multiple_Race": "Multirace",
+                 "Other": "Other_Race", "Unknown": "Unknown_Race"}
+
+
+    # format query to get entry for Alameda County
+    param_list = {'where': "Geography='Alameda County'", 'outFields': '*', 'outSR':4326, 'f':'json'}
+    # get cases data
+    response = requests.get(demographics_cases, params=param_list)
+    response.raise_for_status()
+    parsed = response.json()
+    cases_data = parsed['features'][0]['attributes']
+    # get deaths data
+    response = requests.get(demographics_deaths, params=param_list)
+    response.raise_for_status()
+    parsed = response.json()
+    deaths_data = parsed['features'][0]['attributes']
+   
+    # copy dictionary structure of 'out' dictionary to local variable
+    demo_totals = { "case_totals": out["case_totals"], "death_totals": out["death_totals"]} 
+
+    # Parse and re-key
+    # gender cases and deaths
+    for k, v in GENDER_KEYS.items():
+        demo_totals["case_totals"]["gender"][k] = cases_data[v]
+        if k in deaths_data.keys(): # the deaths table does not currently include MTF or FTM
+            demo_totals["death_totals"]["gender"][k] = deaths_data['Deaths_' + v]
+    # race cases and deaths
+    for k, v in RACE_KEYS.items():
+        demo_totals["case_totals"]["race_eth"][k] = cases_data[v]
+        demo_totals["death_totals"]["race_eth"][k] = deaths_data['Deaths_' + v]
+    # get age cases and deaths
+    demo_totals["case_totals"]["age_group"] = { k: v for k, v in cases_data.items() if 'Age' in k }
+    demo_totals["death_totals"]["age_group"] = {k: v for k, v in deaths_data.items() if 'Age' in k}
+
+    # Handle values equal to '<10', if any. Note that some data points are entered as `null`, which 
+    # will be decoded as Python's `None`
+    counts_lt_10 = []
+    for cat, cat_dict in demo_totals.items():  # cases, deaths
+        for group, group_dict in cat_dict.items():  # dictionaries for age, race/eth
+            for key, val in group_dict.items():
+                if val == '<10':
+                    counts_lt_10.append(f"{cat}.{group}.{key}")
+                elif val!= None: # if the value wasn't null
+                    try:
+                        int(val)
+                    except ValueError:
+                        raise ValueError(f'Non-integer value for {key}')
```

Comment:
Should `None` be converted to `-1` for these? e.g:

```py
elif val is None:
    group_dict[key] = -1
else:
    try:
        int(val)
    except ValueError:
        raise ValueError(f'Non-integer value for {key}')
```

Reply:
(Since I *think* the whole point of using `-1` instead of `null` was to make sure the values are always numbers and ensure easy parsing for strongly typed tools/languages.)